docs/docsite/rst/user_guide/playbooks_loops.rst

```diff
@@ -1,21 +1,40 @@
 .. _playbooks_loops:
 
+*****
 Loops
-=====
+*****
 
-Often you'll want to do many things in one task, such as create a lot of users, install a lot of packages, or
-repeat a polling step until a certain result is reached.
+Sometimes you want to repeat a task multiple times. In computer programming, this is called a loop. Common Ansible loops include installing several packages with the :ref:`yum module <yum_module>`, creating multiple users with the :ref:`user module <user_module>`, and
+repeating a polling step until a certain result is reached. Ansible offers two keywords for creating loops: ``loop`` and ``with_<lookup>``.
 
-This chapter is all about how to use loops in playbooks.
+.. note::
+   * We added ``loop`` in Ansible 2.5, but it is not yet a full replacement for ``with_<lookup>``.
+   * We have not deprecated the use of ``with_<lookup>``.
+   * We are still discussing what UX changes can be made to enable ``loop`` to replace ``with_<lookup>`` in the future.
+   * If we eventually deprecate the ``with_`` syntax, the deprecation cycle will be longer than usual.
```

Comment:
not sure i would put this in docs, some thing more like:

 - loop is preferred, but with_<lookup> will still be valid for the foreseeable future.
 - we are looking to improve ``loop`` syntax, so you imight want to keep a close eye on this page and changelogs in future versions.